Header header
geometry_msgs/Point32[] object_points
camera_calibration/ImagePoint[] image_points
uint8 success
